resources/views/layout.blade.php

```diff
@@ -7,7 +7,7 @@
     <meta name="csrf-token" content="{{ csrf_token() }}">
     <link rel="shortcut icon" href="{{ asset('/vendor/horizon/img/favicon.png') }}">
 
-    <title>Horizon</title>
+    <title>Horizon{{ (config('app.name') ? ' - ' . config('app.name') : '') }}</title>
```

Comment:
You don't need the outer `()` here. 

```suggestion
    <title>Horizon{{ config('app.name') ? ' - ' . config('app.name') : '' }}</title>
```